ydb/docs/en/core/reference/ydb-cli/workload-click-bench.md

```diff
@@ -20,50 +34,75 @@ See the description of the command to init the data load:
 {{ ydb-cli }} workload clickbench init --help
 ```
 
-### Available parameters {#init-options}
+### Available parameters { #init_options }
 
-| Parameter name | Parameter description |
+Option name | Option description
 ---|---
-| `--table <value>` | The table name. The default value is `clickbench/hits`. |
-| `--store <value>` | Type of table storage. Acceptable values: `row`, `column`. Default value: `row`. |
+`--store <value>` | Table storage type. Possible values: `row`, `column`, `external-s3`. Default value is `row`.
+`--external-s3-prefix <value>` | Only relevant for external tables. Root path to the dataset in S3 storage.
+`--external-s3-endpoint <value>` or `-e <value>` | Only relevant for external tables. Link to S3 Bucket with data.
+
+`--string` | Use `String` type for text fields. `Utf8` is used by default.
 
-## Uploading data to the table {#load}
+`--datetime` | Use `Date`, `Datetime` and `Timestamp` type for time-related fields. `Date32`, `Datetime64` and `Timestamp64` are used by default.
 
-Upload data to the table. To do this, download and unzip the data archive, then upload the data to the table:
+`--clear` | If the table at the specified path has already been created, it will be deleted.
+
+## Loading data into a table { #load }
+
+Load data into a table. To do this, download the archive with the data, then load the data into the table:
 
 ```bash
 wget https://datasets.clickhouse.com/hits_compatible/hits.csv.gz
-gzip -d hits.csv.gz
-{{ ydb-cli }} import file csv --path clickbench/hits --input-file hits.csv
+{{ ydb-cli }} workload clickbench --path clickbench/hits import files --input hits.csv.gz
 ```
 
-## Running a load test {#run}
+You can use both unpacked and packed csv and tsv files, as well as directories with such files, as source files.
+
+### Available parameters { #load_files_options }
+
+Option name | Option description
+---|---
+`--input <path>` or `-i <path>` | Path to source data files. Both unpacked and packed csv and tsv files, as well as directories with such files, are supported. The data can be downloaded from the official ClickBench website: [csv.gz](https://datasets.clickhouse.com/hits_compatible/hits.csv.gz), [tsv.gz](https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz). To speed up the download, you can split these files into smaller parts, in which case the parts will be downloaded in parallel.
+`--state <path>` | Path to the download state file. If the download was interrupted for some reason, the download will continue from the same place when restarted.
+`--clear-state` | Relevant if the `--state` parameter is specified. Clear the state file and start the download from the beginning.
```

Comment:
```suggestion
### Available parameters { #load_files_options }

| Name                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `--input <path>` or `-i <path>` | Path to the source data files. Both unpacked and packed CSV and TSV files, as well as directories containing such files, are supported. Data can be downloaded from the official ClickBench website: [csv.gz](https://datasets.clickhouse.com/hits_compatible/hits.csv.gz), [tsv.gz](https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz). To speed up the process, these files can be split into smaller parts, allowing parallel downloads. |
| `--state <path>`           | Path to the download state file. If the download is interrupted, it will resume from the same point when restarted.                                                                                                                                                                                                                                                                                                                                           |
| `--clear-state`            | Relevant if the `--state` parameter is specified. Clears the state file and restarts the download from the beginning.                                                                                                                                                                                                                                                                                                                                          |
```